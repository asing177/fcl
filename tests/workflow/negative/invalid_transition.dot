digraph workflow {
edge [color="#000000"]
graph [bgcolor=transparent]


initial [shape=point; width=0.3; style=filled; color="#0e64ce"; label=""]
bad [shape=ellipse; fontname="Arial"; fontsize=16; style=filled; color=black; fillcolor=white;]
terminal [shape=point; width=0.3; peripheries=2; style=filled; color="#d11010"; label=""]

1[label=<<FONT POINT-SIZE="16">init</FONT><FONT POINT-SIZE="10" COLOR="blue"> </FONT>>shape=box; fontname="Arial"; style=filled; color=black; fillcolor=gray75;]
2[label=<<FONT POINT-SIZE="16">init</FONT><FONT POINT-SIZE="10" COLOR="blue"> </FONT>>shape=box; fontname="Arial"; style=filled; color=black; fillcolor=gray75;]

initial -> 1 ; 1 -> bad
initial -> 2 ; 2 -> terminal



}
