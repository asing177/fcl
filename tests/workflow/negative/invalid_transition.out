Transition error at Line 5, Column 3:
   This statement transitions, leaving the following statement at Line 6, Column 3 unreachable.